### PR TITLE
add ability to add background to gannt group by view

### PR DESCRIPTION
**Pull request by kkarolis**
8361

### CODE REVIEW

**Comment by kkarolis** [comment]
https://hive.versada.eu/work_packages/8361

**Comment by paulius-sladkevicius** [comment]
Looks clean and nice, however I'm poor with JS knowledge, so @taezaz please review.

File: web_timeline/static/src/js/timeline_renderer.js [hunk 1]
```diff
@@ -308,6 +320,30 @@ var CalendarRenderer = AbstractRenderer.extend({
         return r;
     },
 
+    add_group_backgrounds: function(datas, events) {
+        var self = this;
+        var background_data = [];
+        _.each(_.groupBy(datas, 'group'), function(grouped_data, group) {
+            // find related min/max data from events themselves
+            if (group > -1) {
```
Comment (taezaz) [inline web_timeline/static/src/js/timeline_renderer.js]: super nitpick: could use early return instead of nesting

**Review by taezaz — APPROVED** [review]
:tangerine:

**Pull request merged**